### DISCUSSION TITLE
Architecture Documentation

### DISCUSSION THREAD

**Discussion by damienwebdev** [faker-js/faker#747] [Ideas]
Generally speaking, the FakerJs library has been lax about principles and practices about important architecture aspects of the package. 

We should be explicitly clear about exact how we come to decisions about code-related things. This may include, but is not limited to:

1. The fallback mechanism for localization
2. Naming conventions for functions
3. What modules fall into scope of Faker, or what should be instead implemented externally from Faker.

**Comment by ST-DDT** [comment]
+ Versioning and LTS policy

**Reply by Shinigami92** [reply]
Yeah ^^ I would also like to know how we handle this.
Because in yesterday's discussion suddenly it was a different strategy as I understand of what we discussed last week
